code/game/turfs/closed/minerals.dm

```diff
@@ -517,6 +517,11 @@
 	smooth_icon = 'icons/turf/walls/red_wall.dmi'
 	base_icon_state = "red_wall"
 
+/turf/closed/mineral/asteroid/porous
+	name = "porous rock"
+	desc = "This rock is filled with pockets of breathable air."
+	baseturfs = /turf/open/floor/plating/asteroid
```

Comment:
So basically, this is rock when mined, makes red floor full of air?

Reply:
yeah, so like if you put these in maint, you dont want 0 pressure if its mined, youd like it to have a bit of air.